apps/crud/src/app/services/getTodo.service.ts

```diff
@@ -0,0 +1,86 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { TodoConfig } from '../core/Interface/todo';
+import { randText } from '@ngneat/falso';
+import { Observable, of } from 'rxjs';
+import { ToDoStore } from '../store/todo.store';
+@Injectable({
+  providedIn: 'root',
+})
+export class GetToDoService {
+  private commonUrl = 'https://jsonplaceholder.typicode.com/todos';
+  public data$: Observable<TodoConfig[]>;
+  public globalLoader$: Observable<boolean> = of(false);
+  public localLoader$: Observable<boolean> = of(false);
```

Comment:
localLoader$ = this.todoStore.localLoader$ should be done here.

Reply:
done